tests/integration/kubernetes/k8s-runtime-set-policy.bats

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bats
+#
+# Copyright (c) 2023 Microsoft.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+load "${BATS_TEST_DIRNAME}/../../common.bash"
+load "${BATS_TEST_DIRNAME}/tests_common.sh"
+
+setup() {
+	get_pod_config_dir
+	pod_name="runtime-set-policy"
+	pod_yaml="${pod_config_dir}/k8s-runtime-set-policy.yaml"
+	allow_all_rego="${pod_config_dir}/allow-all.rego"
+	reject_exec_rego="${pod_config_dir}/reject-exec.rego"
+}
+
+@test "kubectl exec and logs enable/disable at runtime" {
```

Comment:
@LindaYu17, we have a problem with this test, as we cannot assume we have `kata-runtime` in the same place where we're running the tests.

For instance, we have tests running on AKS, all the deployment is done in the AKS nodes, and kata-policy is never available for us.

I'd recommend we move this test into a unit test (or similar).

Apart from this, the PR looks good.